web/app/view/UsersController.js

```diff
@@ -40,6 +40,12 @@ Ext.define('Traccar.view.UsersController', {
     onAddClick: function () {
         var user, dialog;
         user = Ext.create('Traccar.model.User');
+        if (Traccar.app.getUser().get('admin')) {
+            user.set('deviceLimit', -1);
+        }
+        if (Traccar.app.getUser().get('expirationTime')) {
+            user.set('expirationTime', Traccar.app.getUser().get('expirationTime'));
```

Comment:
Made for convenience, if manager has expiration time, new user will have same expiration time by default.